### PR TITLE
#50 add test for support notifications client

### PR DESCRIPTION
**Pull request by yanghua**
Signed-off-by: yanghua <yanghua1127@gmail.com>

hi @drasko and @feclare , I add a test for support notification client because I find this call : 
https://github.com/edgexfoundry/edgex-go/blob/master/support/notifications-client/client.go#L91

will not trigger a real remote request if it run in a new `goroutine`.

Please review this test, I will try to refactor the notification client and this test code. Thanks!

### CODE REVIEW

File: support/notifications-client/client_test.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
+//
+// Copyright (c) 2017 Tencent
```
Comment (drasko) [inline support/notifications-client/client_test.go]: We're in 2018.

Comment (yanghua) [reply support/notifications-client/client_test.go]: I will change this, I just follow the pre-existing style before.

Comment (yanghua) [reply support/notifications-client/client_test.go]: @drasko changed it.

**Comment by yanghua** [comment]
hi @tsconn23 , if you have time, would you please merge this PR? Because, I will refactor the notification client and use this test case to verify the function. Thanks!

**Pull request merged**